### DISCUSSION TITLE
Missing Padding values

### DISCUSSION THREAD

**Discussion by luantran1311** [tailwindlabs/tailwindcss#11512] [Help]
Hello, do you have any idea why "py-36" does not give element top & bottom paddings? ("p-36" padding works though)
![image](https://github.com/tailwindlabs/tailwindcss.com/assets/19511189/f91e8844-3279-4f6d-b895-73aa22ffe80d)

**Comment by RobinMalfait** [comment]
Hey, thank you for your question! 🙏

There can be multiple problems here, the typical problems are:

1. The content array is not listing all the files you are editing (see: https://tailwindcss.com/docs/content-configuration for more info)
2. You overrode the padding configuration (but since `p-36` _is_ working, I don't think it is this)
3. You forgot to re-run your build process to process the files. (If you setup your project using Vite, Next.js, ... then this shouldn't be an issue)
4. You are creating the value dynamically, for example `p-${paddingValue}` this is something that won't work. (see: https://tailwindcss.com/docs/content-configuration#dynamic-class-names for more info)

We have a section for troubleshooting here with some additional information: https://tailwindcss.com/docs/content-configuration#troubleshooting

If none of these options work for you, can you attach a minimal reproduction repo that shows the problem?

**Comment by roma-marshall** [comment]
Try this `!py-36` or `py-[9rem]`
